src/hotspot/share/gc/serial/serialBlockOffsetTable.hpp

```diff
@@ -39,165 +39,37 @@
 // The CollectedHeap type requires subtypes to implement a method
 // "block_start".  For some subtypes, notably generational
 // systems using card-table-based write barriers, the efficiency of this
-// operation may be important.  Implementations of the "BlockOffsetArray"
+// operation may be important.  Implementations of the BlockOffsetTable
 // class may be useful in providing such efficient implementations.
```

Comment:
The comment is completely outdated, talking about subtypes where there are none any more of this class.

Reply:
I think the subtypes refer to heaps from different collectors. Not sth around `BlockOffsetArray`.

Reply:
OK, removed.